controller/OrderController.go

```diff
@@ -0,0 +1,128 @@
+package controller
+
+import (
+	"fmt"
+	"github.com/Baraulia/COURIER_SERVICE/dao"
+	_ "github.com/Baraulia/COURIER_SERVICE/dao"
+	"github.com/gin-gonic/gin"
+	"log"
+	"net/http"
+	"strconv"
+)
+
+
+
+     var Orders []dao.Order
+func (h *Handler) GetCourierCompletedOrders(c *gin.Context) {
+	page,er:= strconv.Atoi(c.Query("page"))
+	if er!=nil{
+		c.JSON(http.StatusBadRequest, gin.H{"message": "no such page"})
+		return
+	}
+	limit,er1:= strconv.Atoi(c.Query("limit"))
+	if er1!=nil{
+		c.JSON(http.StatusBadRequest, gin.H{"message": "no such limit"})
+		return
+
+	}
+	idCourier,er:= strconv.Atoi(c.Query("idcourier"))
+	if er!=nil{
+		c.JSON(http.StatusBadRequest, gin.H{"message": "no such id"})
+		return
+	}
+	if er==nil && page!=0 && limit!=0{
```

Comment:
you already compared error with `!= nil` and returned from the function, so you don't need to do it once again. As for page <= 0 or limit <= 0 better use `fail fast` approach instead of `if-else`. So similar to how you compared err != nil and immediately returned bad request. Because if there are more conditions like this that also contain different messages you don't end up with a lot of nested if-else